.github/workflows/module_docs_ci.yml

```diff
@@ -2,19 +2,13 @@ name: vlib modules CI
 
 on:
   pull_request:
-    paths-ignore:
-      - '**.vv'
-      - '**.out'
-      - '**.yml'
-      - '**.md'
-      - '!**/module_docs_ci.yml'
+    paths:
+      - 'vlib/**.v'
+      - '**/module_docs_ci.yml'
```

Comment:
Changes to .md files should trigger that too, since they are included in the generation.

Reply:
Yep good point. They were ignored before so I didn't add them.